spec/factories/lectures.rb

```diff
@@ -54,6 +54,21 @@
       end
     end
 
+    trait :with_title do
+      transient do
+        title { nil }
```

Comment:
Form my understanding (which may be wrong), this sets the default value for `title` as `nil`. I would expect that
`course { association :course, title: title }`
will fail if the `title` is `nil` (because of 
`  validates :title, presence: true, uniqueness: true`
in `course.rb`). Is this intended?

Reply:
Yes, this indeed fails when the `title` is `nil` and that was my intention as I wanted to ensure the title is passed in from outside. But I see the point in having a default value for other tests, so will make sure there is one using `Faker`.

Reply:
Ok, so I've added a default title via `Faker::Book` here.